### PR TITLE
Update TransformControls.js - Ability to individually enable/disable the handles/pickers.

### PR DESCRIPTION
**Pull request by jonTrent**
Noob here on GitHub, so my apologies if my etiquette is coarse...

Am proposing changes to TransformControls.js to permit individually disabling / enabling the various handles / pickers, something I noticed others on StackExchange have sought. In short, have added some helper functions to alter the visibility of the meshes / lines that make up the handles / pickers, to permit control over which handles are visible / available for transforming the associated object. Additionally, by adding helper functions, backwards compatibility is maintained.

In short the functions of getControl, setControlAll, and setControl are passed the gizmo ("translate", "rotate", or "scale"), and the handle / picker to enable / disable (eg, "X", "Y", "Z", "XY", "XYZ", etc).  Am open to constructive criticism on this three function API, as I'm not quite sure if there is a defacto function / parameter convention within the THREE framework...

Have updated corresponding TransformControls.html example to show the capabilities of controlling the handles / pickers.

Tested on Chrome, IE11, and Edge.

### CODE REVIEW

**Comment by mrdoob** [comment]
I guess this is related to #12108?

**Comment by jonTrent** [comment]
Yes, this is related to #12108 .  I couldn't quite figure out within web GitHub how to bundle both the TransformControls.js and .html into a single Pull Request...

**Comment by jonTrent** [comment]
I no longer intend to make use of the proposed core changes in this request (and associated #12108), so these two requests can be cancelled / rejected.  (I'm taking a different path by simply writing my own mouse control routines and raycasting to a XY plane in order to translate to appropriate coordinates for my 2d paths...)
